tests/system/providers/google/cloud/kubernetes_engine/example_kubernetes_engine_job.py

```diff
@@ -90,6 +107,15 @@
     )
     # [END howto_operator_gke_describe_job]
 
+    describe_job_task_def = GKEDescribeJobOperator(
```

Comment:
Is this task supposed to be in the deferrable mode?

Reply:
no, it isn't